content/partials/specs/versions-macos-silicon-xcode-16.md

```diff
@@ -0,0 +1,137 @@
+---
+description: A list of tools available out-of-the-box on Codemagic macOS Apple silicon build machines using Xcode version 16.0.
+title: macOS Apple silicon build machine specification (Xcode 16.0+)
+aliases:
+
+weight: 17
```

Comment:
we probably don't need this parameter

Reply:
I'm not sure, just copy-pasted from the former versions. We can remove these in the future updates.